search-app/test/cmr/search/test/api/concepts_search.clj

```diff
@@ -0,0 +1,29 @@
+(ns cmr.search.test.api.concepts_search
```

Comment:
need to rename this namespace (not the file name) to use hyphens rather than underscores. So, change concepts_search to concepts-search

Reply:
(and of course then all other mentions as well)